src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -544,11 +544,7 @@ const Packages = () => {
                       target="_blank"
                       iconPosition="right"
                       icon={<ExternalLinkAltIcon />}
-                      href={
-                        isBeta()
-                          ? '/preview/insights/content'
-                          : '/insights/content'
-                      }
+                      href={BetaPath(CONTENT_URL)}
```

Comment:
Will this work? I know that we were doing the same already, but we have a hook call in a nested inline function.
I'm not sure it's what can do :thinking: